suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -420,7 +420,7 @@ export const networks = {
     sol: {
         name: 'Solana',
         networkType: 'solana',
-        bip43Path: "m/44'/501'/i'", // TODO(vl): accounts and paths
+        bip43Path: "m/44'/501'/i'/0'", // TODO(vl): accounts and paths
```

Comment:
Not much to see here, just updated this because on this path we have an account with balance. Different types of paths will be implemented in future PR.